Mathlib/Algebra/Notation/README.md

```diff
@@ -0,0 +1,7 @@
+# Basic algebraic notation
+
+This subfolder contains definitions and very minimal theory related to the basic algebraic operators
+like `0`, `1`, `+`, `-`, `*`, `/`, `+ᵥ`, `•`, `-ᵥ`.
+
+The files in this subfolder should stay extremely early in the import hierarchy, and in particular
```

Comment:
Can/should this be enforced using a linter? (That should not be hard to write, we have an existing linter that could be extended.)

Reply:
It sadly cannot yet be enforced

Reply:
Oh sorry got confused. This one can be enforced